application/values.yaml

```diff
@@ -59,54 +77,66 @@ deployment:
     #   persistentVolumeClaim:
     #     claimName: claim-name
 
+  # Mount path for Volumes 
+  volumeMounts: []
+    # - mountPath: /path
+    #   name: volume-name
+
+  # Taint tolerations for nodes
   tolerations: []
     # - key: "dedicated"
     #   operator: "Equal"
     #   value: "app"
     #   effect: "NoSchedule" 
 
-
+  # Image of the app container
   image: 
     repository: repository/image-name
     tag: v1.0.0
     pullPolicy: IfNotPresent
 
-  probes: {}
-    # readinessProbe:
-    #   failureThreshold: 3
-    #   periodSeconds: 10
-    #   successThreshold: 1
-    #   timeoutSeconds: 1
-    #   initialDelaySeconds: 10
-    #   httpGet:
-    #     path: /path
-    #     port: port
-    # livenessProbe:
-    #   failureThreshold: 3
-    #   periodSeconds: 10
-    #   successThreshold: 1
-    #   timeoutSeconds: 1
-    #   initialDelaySeconds: 10
-    #   httpGet:
-    #     path: /path
-    #     port: port
+  # Readiness and Liveness probes
+  probes:
+    readinessProbe:
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      httpGet:
+        path: /path
+        port: 8080
+    livenessProbe:
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      httpGet:
+        path: /path
+        port: 8080
+
+  # Resources to be defined for pod
   resources: 
-    # limits:
-    #   memory: "128Mi"
-    #   cpu: "0.5"
-    # requests:
-    #   memory: "64Mi"
-    #   cpu: "0.1"
+    limits:
+      memory: 256Mi
+      cpu: 1
+    requests:
+      memory: 128Mi
+      cpu: 0.5
```

Comment:
Set the resources a bit less like requests to 0.1, limits to 0.5

Reply:
Yes agreed; it should be very small value